common/templates/general.go

```diff
@@ -258,6 +258,10 @@ func CreateMessageSend(values ...interface{}) (*discordgo.MessageSend, error) {
 		case "filename":
 			// Cut the filename to a reasonable length if it's too long
 			filename = common.CutStringShort(ToString(val), 64)
+		case "reply":
+			msg.Reference = &discordgo.MessageReference{
+				MessageID: val.(int64),
```

Comment:
It is easy to cause a panic with this implementation. And the error might look weird to the end user: `error calling complexMessage: interface conversion: interface {} is T, not int64`, T being the type provided.

What if we handle those errors?
```go
case "reply":
	mID := ToInt64(val)
	if mID <= 0 {
		return nil, errors.New("invalid message id provided to reply: " + ToString(val))
	}
	msg.Reference = &discordgo.MessageReference{
		MessageID: mID,
	}
```
This way the error to the user would be: `error calling complexMessage: invalid message id provided to reply: V`, v being the value of whatever they provided. And this is not a panic as well.